src/components/Item/Item.jsx

```diff
@@ -0,0 +1,37 @@
+import styles from "./Item.module.scss";
+import heart from "../../image/heart.svg";
+import ImageWithValidation from "./imageInvalid";
+
+export default function Item({ item, listType }) {
+  const styleClass = `${
+    listType === "best" ? styles["image--inBest"] : styles["image--inAll"]
+  } ${item.images.length === 0 ? styles.noImage : ""}`;
+
+  return (
+    <div className={styles.item}>
+      {item.images ? (
+        <ImageWithValidation
+          styleClass={styleClass}
+          imageURL={item.images}
+          alt="프로필 이미지"
+        />
+      ) : (
+        <div className={styleClass}></div>
+      )}
+      {/* //   <ImageWithValidation
+    //     styleClass={styleClass}
+    //     imageURL={item.images}
+    //     alt="프로필 이미지"
+    //   /> */}
+
+      <div className={styles.item__description}>
+        <h3 className={styles["item__name"]}>{item.name}</h3>
+        <p className={styles["item__price"]}>{item.price}</p>
```

Comment:
💊 제안
금액의 경우 (,)를 통해 단위가 구분되도록 아래처럼 작성해주세요!
```suggestion
        <p className={styles["item__price"]}>{item.price.toLocaleString()}</p>
```